### PR TITLE
Fixed hard-coded and hard to read colors in ServerBrowser

### LINKED ISSUE #7901: "Password protected" font color in RA server browser is unreadable.
This should be changed to a different shade of red.  Ideally this colour should be customizable per-mod via ChromeMetrics.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/10698032-password-protected-font-color-in-ra-server-browser-is-unreadable?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Mailaender**
Made the server browser text color per-mod configurable and fixed https://github.com/OpenRA/OpenRA/issues/7901.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Looks good to me :+1: / :white_check_mark:

File: OpenRA.Mods.Common/Widgets/Logic/ServerBrowserLogic.cs [hunk 1]
```diff
@@ -217,7 +217,8 @@ void RefreshServerListInner(IEnumerable<GameServer> games)
 					if (title != null)
 					{
 						title.GetText = () => game.Name;
-						title.GetColor = () => !compatible ? Color.DarkGray : !canJoin ? Color.LightGray : title.TextColor;
+						title.GetColor = () => !compatible ? ChromeMetrics.Get<Color>("IncompatibleGameColor")
```
Comment (obrakmann) [inline OpenRA.Mods.Common/Widgets/Logic/ServerBrowserLogic.cs]: Can you please cache those lookups?

**Comment by obrakmann** [comment]
Looks otherwise fine to me.

File: OpenRA.Mods.Common/Widgets/Logic/ServerBrowserLogic.cs (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/ServerBrowserLogic.cs]: Good idea. Done.

**Comment by obrakmann** [comment]
Thanks :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/2ef98a0bedc0ddffacc146711468e4e6b5295544%5E...2ef98a0bedc0ddffacc146711468e4e6b5295544)

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/2ef98a0bedc0ddffacc146711468e4e6b5295544...abe4fd4bd4937cb52a2f1a1f4421336f10abef5e)

**Pull request merged**